docs/configuration.md

```diff
@@ -69,6 +67,11 @@ Unless you need to set a non-default value, it is recommended to only populate o
 | `ETH_FEE_HISTORY_FIXED`                     | "true"                   | Flag to set if eth_feeHistory should return a fixed fee for the set of results.                                                                                                                                                                                                                        |
 | `GAS_PRICE_PERCENTAGE_BUFFER`               | "0"                      | The additional buffer that adds a percentage on top of the calculated network gasPrice. This may be used by operators to reduce the chances of `INSUFFICIENT_TX_FEE` errors experienced by users caused by minor fluctuations in the exchange rate.                                                    |
 | `GAS_PRICE_TINY_BAR_BUFFER`                 | "10000000000"            | The additional buffer range to allow during a relay precheck of gas price. This supports slight fluctuations in network gasprice calculations.                                                                                                                                                         |
+| `HBAR_RATE_LIMIT_DURATION`                  | "80000"                  | HBar budget limit duration. This creates a timestamp, which resets all limits, when it's reached. Default is to 80000 (80 seconds).                                                                                                                                                                    |
+| `HBAR_RATE_LIMIT_TINYBAR`                   | "11_000_000_000"         | Total hbar budget in tinybars (110 hbars per 80 seconds).                                                                                                                                                                                                                                              |
```

Comment:
Nit, no need to repeat 80 seconds. If anything note `HBAR_RATE_LIMIT_DURATION` so that the text isn't prone to being outdated